### PR TITLE
Add analytics to example app

### PR DESCRIPTION
**Pull request by eessex**
Follow up to web-practice work and #6711, adds analytics to v2 example app, and a few components that will fire events.

### CODE REVIEW

File: src/v2/Apps/Consign/Routes/MarketingLanding/MarketingLandingApp.tsx [hunk 1]
```diff
@@ -19,35 +18,28 @@ import { TemporaryOffer } from "./Components/TemporaryOffer"
 
 export const MarketingLandingApp = () => {
   return (
-    <AnalyticsContext.Provider
```
Comment (eessex) [inline src/v2/Apps/Consign/Routes/MarketingLanding/MarketingLandingApp.tsx:22]: These properties are already available via boot.

File: src/v2/Apps/Fair/FairApp.tsx [hunk 2]
```diff
@@ -175,7 +175,7 @@ const TrackingWrappedFairApp: React.FC<FairAppProps> = props => {
     <AnalyticsContext.Provider
       value={{
         contextPageOwnerId: internalID,
-        contextPageOwnerSlug,
+        contextPageOwnerSlug: slug,
```
Comment (eessex) [inline src/v2/Apps/Fair/FairApp.tsx:178]: Unrelated to example, but adds correct slug to fair analytics.

File: src/v2/Apps/Example/Routes/ExampleArtist/ExampleArtistApp.tsx [hunk 3]
```diff
@@ -15,18 +18,49 @@ const ExampleArtistApp: React.FC<ExampleArtistAppProps> = ({ artist }) => {
...
+        contextModule={ContextModule.artistHeader}
+      />
       <Text>{artist.bio}</Text>
     </Box>
   )
 }
 
+/**
+ * Routes with /:id require an additional AnalyticsContext.Provider
+ * declaration to add slug and id, extending the context provided by <Boot>
```
Comment (damassi) [inline src/v2/Apps/Example/Routes/ExampleArtist/ExampleArtistApp.tsx:32]: This is good to know, i wasn't aware 👍

Comment (eessex) [reply src/v2/Apps/Example/Routes/ExampleArtist/ExampleArtistApp.tsx:32]: Had tried to get this out of the routes, but is a bit trickier than i had hoped - not the ideal solution but it works

**Pull request merged**